contribute/tools.md

```diff
@@ -1,8 +1,123 @@
+<a href="https://github.com/typesafehub/dbuild"><img src="https://avatars3.githubusercontent.com/u/784923?v=3&s=200" width="50px" height="50px"/><h4>DBuild</h4></a>
+<p>Multi-project build tool.</p>
+<p><a href="http://typesafehub.github.io/dbuild">Home</a> | 
+<a href="https://github.com/typesafehub/dbuild/issues">Issues</a> | 
+<a href="https://github.com/typesafehub/dbuild/blob/master/README.md">ReadMe</a> | 
+<a href="https://github.com/typesafehub/dbuild/blob/master/README.md#get-involved">Contributing</a></p>
+</div>
+</div>
+
+<div class="row">
+<div class="span4 doc-block">
+<a href="https://github.com/ensime/ensime-server"><img src="https://avatars0.githubusercontent.com/u/5089042?v=3&s=200" width="50px" height="50px"/><h4>Ensime</h4></a>
+<p>Scala Support for Text Editors</p>
+<p><a href="http://ensime.github.io">Home</a> | 
+<a href="https://github.com/ensime/ensime-server/issues">Issues</a> | 
+<a href="https://github.com/ensime/ensime-server/blob/master/README.md">ReadMe</a> | 
+<a href="https://github.com/ensime/ensime-server/blob/master/README.md#contributions">Contributing</a></p>
+</div>
+<div class="span4 doc-block">
+<a href="https://github.com/scala/scala-abide"><img src="https://avatars1.githubusercontent.com/u/57059?v=3&s=200" width="50px" height="50px"/>
+<h4>Abide</h4></a>
+<p>Lint tooling for Scala</p>
+<p><a href="https://github.com/scala/scala-abide#abide--lint-tooling-for-scala">Home</a> | 
+<a href="https://github.com/scala/scala-abide/issues">Issues</a> | 
+<a href="https://github.com/scala/scala-abide/blob/master/README.md">ReadMe</a> | 
+<a href="https://github.com/scala/scala-abide#extending-abide">Contributing</a></p>
+</div>
+</div>
+
+<div class="row">
+<div class="span4 doc-block">
+<a href="https://github.com/scala/scala-partest"><img src="https://avatars1.githubusercontent.com/u/57059?v=3&s=200" width="50px" height="50px"/>
+<h4>Partest</h4></a>
+<p>Scala Compiler/Library Testing</p>
+<p><a href="http://docs.scala-lang.org/tutorials/partest-guide.html">Home</a> | 
+<a href="https://github.com/scala/scala-partest/issues">Issues</a> | 
+<a href="https://github.com/scala/scala-partest/blob/master/README.md">ReadMe</a></p>
+</div>
+<div class="span4 doc-block">
+<a href="https://github.com/scoverage/scalac-scoverage-plugin"><img src="https://avatars1.githubusercontent.com/u/5998302?v=3&amp;s=200" width="50px" height="50px"/><h4>Scoverage</h4></a>
+<p>Scala code coverage tool</p>
+<p><a href="http://scoverage.org">Home</a> | 
+<a href="https://github.com/scoverage/scalac-scoverage-plugin/issues">Issues</a> | 
+<a href="https://github.com/scoverage/scalac-scoverage-plugin/blob/master/README.md">ReadMe</a> | 
+<a href="https://groups.google.com/forum/#!forum/scala-code-coverage-tool">Contributing</a></p>
+</div>
+</div>
+</div>
+
+### Projects in Particular Need
+
+The following projects are important to the Scala community but are particularly in need of contributors to continue their development.
```

Comment:
A compromise - I feel the current order reflects (or ought to) the importance to the scala ecosystem of these projects right now (although, maybe weigh in with your opinions on that), but instead I will put a link up at the top pointing down the page to the projects in particular need. It's a halfway solution, but I think it might work. I also shuffled the order of the projects to be sbt, scaladoc, scala IDE, dbuild, partest, scoverage and abide in that order, if anyone feels a different priority is warranted, let me know (but the icons also kind of work like that too).